### ISSUE TITLE
yum heart cannot stop the bleeding of shard of glass when at full health

### ISSUE THREAD

**Issue by BadsyEdgecat** [UnderscoreKilburn/repentance-issues#1434]

**Comment by ZMBanshee** [comment]
I don't think this is necessarily a bug, I mean it's kind of like how hearts on the ground don't work either when you're at full health.

That said, it would be a nice quality of life change if it allowed healing effects at full health specifically to remove the bleed effect (including the ability to take hearts).
